go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/go-gl/mathgl
+
+go 1.11
```

Comment:
> Support go1.11 onwards.

I want to point out that this directive means to use the Go language version 1.11; it’s not for setting the minimum version of Go that is supported. Letting you know in case you misunderstood this directive.

It’s not currently well documented and that’s issue https://github.com/golang/go/issues/30791.

It’s fine for us to use the the Go language version 1.11, so this value is not a problem.

Reply:
Thanks for reviewing this and thanks for your comment. I almost certainly don't understand it! Will try and find a moment to do so.

Reply:
Found a moment to read the thread and see that it's subtle. What's lost on me is how you could tell that I didn't know, and what the implications of choosing 1.11 rather than 1.12 are! Would appreciate if you can share.

Reply:
> What's lost on me is how you could tell that I didn't know

It was a guess based on that you picked version 1.11 rather than 1.12, given we're trying to support Go 1.12 and 1.11. A common misconception is that this needs to be set to the oldest version of Go that a project supports.

> and what the implications of choosing 1.11 rather than 1.12 are

The Go language has no changes between 1.11 and 1.12, so there are no implications at this time.